onnxruntime/python/tools/symbolic_shape_infer.py

```diff
@@ -548,7 +548,13 @@ def _onnx_infer_subgraph(self, node, subgraph, use_node_input=True, inc_subgraph
         self.symbolic_dims_.update(new_dims)
         return symbolic_shape_inference
 
-    def _get_int_values(self, node, broadcast=False):
+    def _get_int_values(self, node, broadcast=False, loose_int_with_decimal_part=False):
```

Comment:
rename to _get_int_or_float_values(seld, node, broadcast=False, allow_float_values=False)

Reply:
Hi Tianlei your suggestion has been applied to the PR. Thanks!